tools/firmware.toit

```diff
@@ -700,32 +700,32 @@ extract-host parsed/cli.Parsed envelope/Envelope --config-encoded/ByteArray:
 
   // Reserve space for the header.
   header-size := 6 * 4
-  bits.reserve header-size
+  bits.grow-by header-size
```

Comment:
expand?

Reply:
I think "grow-by" is fine, and I would prefer not to change the `io.Writer`.